src/components/SearchableSelect.tsx

```
@@ -91,58 +92,93 @@ const SearchableSelect = <Option, IsMulti extends boolean = false, Group extends
-              return '!border !border-danger-500';
+              return twMerge( '!border !ring-danger-500 !ring-1', userControlStyles );
             }
 
-            return '!border-gray-200';
+            return twMerge( '!border-gray-200', userControlStyles );
           },
         }}
         styles={{
-          control: (base) => {
+          // Pass through any container-level custom styles if provided
+          ...(props?.styles ? (props.styles as any) : {}),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)